orga/app/components/banner/information.hbs

```diff
@@ -5,14 +5,10 @@
       @options={{hash
         faIcon=(component 'fa-icon' icon='external-link-alt')
         linkTo=(component 'link-to' (t 'banners.import.link-to') 'authenticated.sco-students' class="link link--banner link--bold link--underlined")
-        middleSchoolDocumentationLink='"https://view.genial.ly/5f295b80302a810d2ff9fa60/?idSlide=cd748a12-ef8e-4683-8139-eb851bd0eb23"'
-        highSchoolDocumentationLink='"https://view.genial.ly/5f46390591252c0d5246bb63/?idSlide=cd748a12-ef8e-4683-8139-eb851bd0eb23"'
+        middleSchoolDocumentationLink=this.middleSchoolDocumentationLink
+        highSchoolDocumentationLink=this.highSchoolDocumentationLink
```

Comment:
**suggestion (not-blocking)**: être plus spécifique car sur l'autre bannière, il va y avoir des liens de ce type aussi. `importMiddleSchoolDocumentationLink` et `importHighSchoolDocumentationLink` par exemple

Reply:
👍